src/ucp/wireup/wireup.c

```diff
@@ -145,6 +147,29 @@ ucs_status_t ucp_wireup_msg_progress(uct_pending_req_t *self)
     wireup_msg_iov[1].iov_base = req->send.buffer;
     wireup_msg_iov[1].iov_len  = req->send.length;
 
+    /* bcopy size may by limited by max_bcopy capability, make sure we have
+     * enough space.
+     * TODO: find reliable way of exchange wireup message without this limit */
+    packed_len = wireup_msg_iov[0].iov_len + wireup_msg_iov[1].iov_len;
+    uct_ep     = ucp_ep_get_lane(ep, req->send.lane);
```

Comment:
can use ucp_ep_get_rsc_index and ucp_wireup_ep_get_aux_rsc_index get rsc_index of the wireup transports, then use  ucp_worker_iface_get_attr to get max bcopy